### ISSUE TITLE
Spec is missing stub adapter for modal windows

### ISSUE THREAD

**Issue by pavel-krivanek** [pharo-spec/Spec#238]
Spec is missing stub adapter for modal window and other similar presenters like dialog window, ticking window etc.

In addition, we should have some registry for the stub windows (of any kind) to be able to write tests for multi-window applications. Maybe we should have in Spec a concept of an `application` that should do it. 

It is related to issue #235 because such application registry of windows is needed for it.

**Comment by estebanlm** [comment]
what are: 

- MorphicWindowAdapter
- MorphicDialogWindowAdapter
- MorphicModalWindowAdapter
- MorphicTickingWindowAdapter

?
(and btw, I think two of those should gone)

**Comment by pavel-krivanek** [comment]
yes, but we have only the `SpecStubWindowView` so if you want to write tests for application with modal windows, you cannot. Generally, Stub adapters must correspond to the Morphic ones.

**Comment by pavel-krivanek** [comment]
`MorphicTickingWindowAdapter` is used for `TickingWindowPresenter` which says: "I provide an API to execute the #step method every #stepTime."

**Comment by estebanlm** [comment]
I want to drop at least `MorphicTickingWindowAdapter` (this is complete useless... instead we can have an alarm mechanism which can be based on stepping but no need to have a window dedicated to it).
I want also to merge Dialog and Modal (they are there now just for backward compatibility purposes)

**Comment by jecisc** [comment]
We should see what we will do with Modal and Dialogs. Should we keep both of them?
